server/testutils/mmcontainer/testscontainer.go

```diff
@@ -0,0 +1,359 @@
+package mmcontainer
```

Comment:
And ths is the implementation of the testcontainers module for mattermost, this should be usable for other plugins, maybe it should be as part of the started template or the public mattermost libraries, or another repository. I don't know, but this can be very useful for other plugins testing.